### ISSUE TITLE
Syntax highlighting

### ISSUE THREAD

**Issue by szs8** [geocar/q-mode#1]
Any plans for syntax highlighting?

Thanks.

**Comment by geocar** [comment]
I wasn't planning on it. I implemented bracket flashing since I like that, but I usually disable syntax highlighting :)

You might be able to steal bits from http://code.kx.com/svn/contrib/ashih/emacs/kdbp-mode.el

**Comment by szs8** [comment]
Fair enough. There are a bunch of these KDB+ emacs modes lying around and I am unsure which one is the best/most updated.

**Comment by geocar** [comment]
They do different things. This one makes it easy for me to do interactive development (with its repl/notebook and "send-to-kdb" operations). kdbp just does syntax highlighting and doesn't speak to a remote KDB process. I see no reason they couldn't be merged, but I don't use/like syntax highlighting so I didn't implement it.
